internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -678,3 +685,27 @@ func shouldTrackChange(updates ConfigTableUpdates, entityJSON string) bool {
 	}
 	return !updates.DisableChangeTracking(entityJSON)
 }
+
+// fieldIsEmpty checks if a given field is one of:
+//   - Object with no nested items
+//   - Type is null
+//   - Zero length array
+//   - Zero length string
```

Comment:
```suggestion
//   - String is empty
```
Technically, you're not checking the length